gcovr/templates/base.html

```diff
@@ -16,21 +16,29 @@
   </head>
 
   <body>
-    <h1>{{info.head}}</h1>
+    <header>
+      <h1>{{info.head}}</h1>
 
-    <hr/>
+      <hr/>
 
-    <div class="summary">
-      {% block summary %}
-      {% endblock %}
-    </div>
+      <div class="summary">
+        {% block summary %}
+        {% endblock %}
+      </div>
 
-    <hr/>
+      <hr/>
+    </header>
 
-    {% block content %}
-    {% endblock %}
+    <navigation>
+      {% block navigation %}
+      {% endblock %}
+    </navigation>
```

Comment:
There is no `<navigation>` element, but there is a `<nav>` (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)